common/constants.go

```diff
@@ -191,6 +191,7 @@ const (
 	PushEvent    = "ds-pushevent"   //query string to specify if an event should be pushed to the EdgeX system
 	ReturnEvent  = "ds-returnevent" //query string to specify if an event should be returned from device service
 	RegexCommand = "ds-regexcmd"    //query string to specify if the command name is in regular expression format
+	Parent       = "descendantsOf"  //Limit returned devices to those who have parent, grandparent, etc. of the given device name
```

Comment:
since there is no namespace for the constant, we need to make the constant name understandable.
please rename to `DecendantsOf` or `ParentQueryParam`
according to other naming convention, I would suggest using `DecendantsOf`

Reply:
Done - also my team pointed out a maxLevels query parameter would be useful, OK to add that at this late stage?

Reply:
Yeah, I agree with this, please add it